pages/404.tsx

```diff
@@ -0,0 +1,57 @@
+import { GetStaticProps } from 'next'
```

Comment:
> Kat, a lot of this feels pretty redundant to the index page - whats happening here?

SO Next.js has a strong opinion that the 404 page should be rendered at build time, which is reasonable! its always identical. But our header & footer are loaded on the main page server side. This is annoying in redundancy and I bet we can extract it out somehow, but I think this redundancy is the least-evil option for go-live

Reply:
See https://nextjs.org/docs/advanced-features/custom-error-page#404-page

Reply:
Sounds good for launch. I've opened a low priority reminder ticket about this: https://github.com/cagov/ui-claim-tracker/issues/457